features/projects/list_projects.feature

```diff
@@ -60,29 +59,24 @@ Feature: Browse and create projects
     When I am on the "projects" page
     Then I should not see the very stylish "New Project" button
 
-#  Scenario: See codeclimate stats
-#    Given I am on the "projects" page
-#    Then I should see a GPA of "3.5" for "hello world"
-
-# This scenario is no longer valid after we added ordering by status and commit count
-#  Scenario: Alphabetically display pagination in "Our Projects" page
-#    Given I am on the "home" page
-#    When I follow "Projects" within the navbar
-#    Then I should see:
-#      | greetings aliens        |
-#      | greetings jupiter folks |
-#      | greetings mercury folks |
-#      | greetings saturn folks  |
-#      | greetings sun folks     |
-#    And I should not see "greetings earthlings"
-#    When I go to the next page
-#    Then I should not see:
-#      | greetings aliens        |
-#      | greetings jupiter folks |
-#      | greetings mercury folks |
-#      | greetings saturn folks  |
-#      | greetings sun folks     |
-#    And I should see "greetings earthlings"
+  Scenario: Ordered display pagination in "Our Projects" page
```

Comment:
great job getting this working, although would be good if we could express the intent, e.g. that we want to see the most recently updated projects at the top ...